api/v1beta1/nodemodulesconfig_types.go

```diff
@@ -27,8 +27,8 @@ type ModuleConfig struct {
 	// When InsecurePull is true, the container image can be pulled without TLS.
 	InsecurePull bool `json:"insecurePull"`
 	//+optional
-	InTreeModuleToRemove string       `json:"inTreeModuleToRemove,omitempty"`
-	Modprobe             ModprobeSpec `json:"modprobe"`
+	InTreeModulesToRemove []string     `json:"inTreeModulesToRemove,omitempty"`
```

Comment:
Wondering how that will work during upgrade. Maybe we need a conversion webhook for NMC actually.

Reply:
I think it should not be an issue: InTreeRemoval is optional so when the CRD is updated, NMC is created/updated or worker tried to load NMC it will do it without a problem. what we need to do is maybe add a code to the worker to use InTreeModule instead of InTreeModules in case InTreeModule is not empty, but InTreeModules is.